sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Security/Ucfg/UniversalControlFlowGraphBuilder.cs

```diff
@@ -334,6 +370,40 @@ private Instruction CreateInstruction(SyntaxNode syntaxNode, string methodId, st
 
             private ISymbol GetSymbol(SyntaxNode syntaxNode) =>
                 semanticModel.GetSymbolInfo(syntaxNode).Symbol;
+
+            public void CreateAttributeInstructions(IParameterSymbol parameter)
+            {
+                foreach (var attribute in parameter.GetAttributes().Where(a => a.AttributeConstructor != null))
+                {
+                    var attributeVariable = CreateTempVariable();
+
+                    CreateInstruction(
```

Comment:
A comment here explaining what it looks like would be handy because these 2 instructions are not easy to read when you don't know precisely what to expect.
Here is my guess:

```
%x = AttributeMethodId
parameterName = %x
```

Am I correct?

Reply:
On one hand, it would be easier to just add the comment here and not waste time in discussions with marginal value. Simple _blame_ would usually reveal the ticket and the example inside it...
On the other hand, if the CreateInstruction calls are difficult to read in all other places, perhaps we need to add comments there too. Do you think this is the case?

Reply:
4th hand: The unit test cases should make it obvious as to what is expected